openrtb_ext/imp_intertech.go

```diff
@@ -0,0 +1,9 @@
+package openrtb_ext
+
+type ExtImpIntertech struct {
+	PlacementID string `json:"placement_id"`
+	// Deprecated: in favor of `PlacementID`
```

Comment:
Are you saying `PageID` and `ImpID` are deprecated in favor of `PlacementID`? The code only ever references the other two and does not consider `PlacementID`

Reply:
Ah good catch. `placement_id` isn't a valid field. It was supposed to have been removed from this struct.